src/storage/renderer/image.ts

```diff
@@ -170,6 +207,15 @@ export class ImageRenderer extends Renderer {
         case 'quality':
           all.quality = parseInt(value, 10)
           break
+        case 'gravity':
+          all.gravity = value
+          break
+        case 'x_offset':
+          all.x_offset = parseFloat(value)
```

Comment:
i think this wouldn't work since currently, we are splitting on `:` to get the transformation type and value, however, gravity has the following format: `gravity:sm:100:100`

So value would just be `sm` in this case. We would need to change slightly the way we parse the string for the gravity

Reply:
not sure i got you, from what i understood we parse string of our own format, isn't it?

```
this.transformOptions = params.reduce((all, param) => {
const [name, value] = param.split(':') as [keyof TransformOptions, any]
```
And TransformOptions is our own interface where we specify `gravity`, `x_offset`, `y_offset`. And then it is applied be method above.

Or do we rely on imgproxy native format here?

Reply:
Ah! You are right!
Ignore the comment above